Scoop/Controller/Onboarding/ProfilePictureViewController.swift

```diff
@@ -92,15 +95,19 @@ class ProfilePictureViewController: OnboardingViewController {
                 print("\(user.firstName) has been updated")
                 self.dismiss(animated: true)
                 self.containerDelegate?.dismiss(animated: true)
-                NetworkManager.shared.profileDelegate?.updateProfile()
+                NetworkManager.shared.profileDelegate?.updateUserProfile()
+                self.uploadButton.backgroundColor = .scoopDarkGreen
+                self.loadingSpinner.stopAnimating()
```

Comment:
Create a weak reference to `self` in this networking closure and unwrap a strong reference.